charts/generic-service/README.md

```diff
@@ -154,3 +154,29 @@ generic-service:
   ingress:
     host: project-name-dev.hmpps.service.justice.gov.uk
 ```
+
+### Postgres database restore cronjob
+Setting 
+```yaml
+---
+postgresDatabaseRestore:
+   enabled: true
+```
+in your `values-prod.yaml`
+will create a scheduled job runs every four hours in production only.  This checks to see if there is a newer version of 
+the NOMIS database since the last database restore and if so then does another restore.  The pre-production credentials
+should be injected into the production namespace and all should then be added as a `namespace_secrets:` section,
+see the `values.yaml` in this repository for an example.
```

Comment:
Maybe link to an example of adding the pre-prod secrets to a prod env in `cloud-platform-environments`?

Reply:
will do